mix.exs

```diff
@@ -49,7 +49,7 @@ defmodule Wallaby.Mixfile do
 
   defp deps do
     [
-      {:httpoison, "~> 0.12"},
+      {:httpoison, "~> 1.1"},
```

Comment:
I think the best for now where to use `"~> 0.12 or ~> 1.0"`, this should work with both while also not breaking the ugly way after a breaking change in HTTPoison 2.0 happens.